slsim/lens.py

```diff
@@ -361,7 +361,7 @@ def los_linear_distortions(self):
             gamma2 = gamma * np.sin(2 * phi)
             self._gamma = [gamma1, gamma2]
         if not hasattr(self, "_kappa"):
-            self._kappa = np.random.normal(loc=0, scale=0.05)
+            self._kappa = abs(np.random.normal(loc=0, scale=0.05))
```

Comment:
kappa really has to go also negative. The average convergence must be =0 in the universe

Reply:
Avoided test for now.